src/frontend/src/modals/shareModal/index.tsx

```diff
@@ -63,6 +67,47 @@ export default function ShareModal({
   const saveFlow = useFlowsManagerStore((state) => state.saveFlow);
   const { data, isFetching } = useGetTagsQuery();
 
+  function successShare() {
```

Comment:
is there any reason to save the flow again after sharing? I saw in the file history that it's already like that, but I think this code was wrong before and we could remove it in this refactor

Reply:
I think it saves the flow immediately after sharing, so it already appears in the sidebar. This way, the user doesn't have to go to the store to find the flow in the 'Saved' section. but ill check